src/Apps/Artwork/Components/PricingContextModal.tsx

```diff
@@ -61,12 +68,25 @@ export class PricingContextModal extends React.Component<State> {
           </Serif>
           <Spacer mt={2} />
         </Modal>
-        <QuestionCircleIcon
-          ml="5px"
-          width="15px"
-          onClick={this.openModal.bind(this)}
-        />
+        <Tooltip width={73} size="sm" content="Learn more">
+          <StyledQuestionCircleIcon
+            ml="5px"
+            width="15px"
+            onClick={this.openModal.bind(this)}
+          />
+        </Tooltip>
       </>
     )
   }
 }
+
+const StyledQuestionCircleIcon = styled(QuestionCircleIcon)`
+  fill: ${color("black60")};
+
+  &:hover {
```

Comment:
Not an issue with this PR but it might be cool to add `hoverFill` as an option to all of our icons! cc @jonallured

Reply:
I think that's a great idea!